docs/data-sources/virtual_go_repository.md

```diff
@@ -0,0 +1,28 @@
+---
+subcategory: "Virtual Repositories"
+---
+# Artifactory Virtual Go Repository Data Source
+
+Retrieves a remote Go repository.
```

Comment:
s/remote/virtual

Looks like this needs to be changed in all the virtual md files.

Reply:
Missed that one, fixed.